assignments/assignment-6/public/scripts.js

```diff
@@ -204,6 +228,49 @@ $(document).ready(function() {
       loadAdminQuestionForm(pointValue);
     }
   });
+  
+  $(".question-point-block").click(function() {
```

Comment:
Let's add these event listeners (from here and down to line 273) into the document.ready() method.

Reply:
Oh i see you already did that. The tabbing threw me off.